README.md

```diff
@@ -1,92 +1,74 @@
-<h1 align="center">Nimiq Validators</h1>
-
-<p align="center">
-  <a href="https://validators-api-mainnet.pages.dev">
-    <img src="./public/favicon.svg" alt="Nimiq Validators" width="64" />
+  <a id="README" href="#README" href="https://github.com/nimiq/core-rs-albatross/blob/albatross/README.md">
+    <img src="https://raw.githubusercontent.com/nimiq/developer-center/refs/heads/main/assets/images/logos/validators-API.svg" alt="Nimiq PoS Albatross Repository" width="600" />
   </a>
+</br>
+</br>
+
+[![Sync Mainnet](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml) [![Sync Testnet](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml)
 
-<p align="center">
-  Details of validators in the Nimiq Blockchain and their scores, calculated using Nimiq's VTS algorithm.
-<p>
+The Nimiq Validators API enables staking pools, based on single validators, to integrate with the Nimiq Wallet. This helps stakers make informed decisions when choosing where to stake their funds.
 
-[![Sync Mainnet](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-mainnet.yml)
-[![Sync Testnet](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml/badge.svg)](https://github.com/nimiq/validators-api/actions/workflows/sync-testnet.yml)
+**Validators and Staking Pools**:
+A validator can operate as a staking pool, allowing multiple users to stake. Pools must provide detailed information such as fees, payout schedules, and contact details to ensure trust and transparency.
 
-## Add your validator information
+Stakers can evaluate a validator’s reliability using the Validator Trustscore (VTS) and review staking pool details like payout schedules to select the best option.
```

Comment:
Do we go for Validator Trustscore or just Validator Score?

Reply:
Trustscore